README.md

```diff
@@ -82,11 +117,11 @@ from laces.components import Component
 
 
 class WelcomePanel(Component):
-    def render_html(self, parent_context):
-        return format_html("<h1>{}</h1>", "Welcome to my app!")
+    def render_html(self, parent_context=None):
+        return format_html("<h1>Welcome to my app!</h1>")
 ```
 
-### Passing context to the template
+### Passing context to the component template
```

Comment:
I have "renamed" a couple of "files" and variables in the examples to make the differences clear between what is part of the component and what is the "parent" view that uses the component.